### ISSUE TITLE
Mac OSX support

### ISSUE THREAD

**Issue by tdegrunt** [m-ou-se/teck-programmer#1]
Hi,

Since I'm using Mac I wondered why it's not working on the Mac.
I read this topic: https://github.com/nonolith/node-usb/issues/30 and removed interface.claim(); from programmer.js, that at least gives me the following output, suggesting it works:

```
❯ teck-firmware-upgrade TrulyErgonomic_v4_00.hex 
Found device in normal mode on bus 29, device 10.
Switching to program mode.
Rescanning for devices in one second.
Found device in program mode on bus 29, device 10.
Uploading firmware.
Uploading done.
Switching back to normal mode.
Done.

```

Any hints? It would be really useful to get this to work on Mac as well...

**Comment by m-ou-se** [comment]
It seems to work just fine on Mac when leaving out the "interface.claim()" line. I wrapped that line in a try-catch statement, so it doesn't stop when it fails.

I'm surprised it works, because I understood that Mac OS doesn't allow raw usb access to HID devices like keyboards, but apparently this information was incorrect or incomplete.

**Comment by m-ou-se** [comment]
Just tested it on a Mac, and uploading new firmware works.

**Comment by m-ou-se** [comment]
Fixed in d4c34f00.

**Comment by tdegrunt** [comment]
Ah indeed, I missed the Layers DIP #1 setting and messed that up.
Glad it works, thank you!
